database/metadata/databases/default/tables/public_crashes_list_view.yaml

```diff
@@ -68,57 +78,57 @@ select_permissions:
         - rpt_street_name
         - rpt_street_pfx
         - rpt_street_sfx
-        - rr_relat_fl
-        - schl_bus_fl
-        - sus_serious_injry_count
-        - toll_road_fl
-        - tot_injry_count
-        - traffic_cntl_id
-        - unkn_injry_count
-        - vz_fatality_count
-        - wthr_cond_id
-        - years_of_life_lost
+        - crash_timestamp
       filter: {}
-      allow_aggregations: true
```

Comment:
@roseeichelmann would you restore these aggregation permissions? then this is good to go